projects/extension/src/background/index.ts

```diff
@@ -45,10 +51,24 @@ const init = async () => {
       const rpcCallback = (rpc: string) => {
         console.warn(`Got RPC from ${key} dummy chain: ${rpc}`)
       }
-      await manager
-        .addChain(value, rpcCallback)
+      const chain = await manager
+        .addChain(value, rpcCallback, undefined, key)
         .catch((err) => l.error("Error", err))
```

Comment:
I don't know how this `catch` got here, but this `catch` shouldn't be here at all.